docs/ru/sql-reference/table-functions/postgresql.md

```diff
@@ -34,6 +34,7 @@ SELECT запросы на стороне PostgreSQL выполняются ка
 INSERT запросы на стороне PostgreSQL выполняются как `COPY "table_name" (field1, field2, ... fieldN) FROM STDIN` внутри PostgreSQL транзакции с автоматическим коммитом после каждого `INSERT` запроса.
 
 PostgreSQL массивы конвертируются в массивы ClickHouse.
+Будьте осторожны в PostgreSQL многомерные массивы могут содержать в себе другие массивы с разным кол-вом элементов в разных подмассивах, но внутри ClickHouse допустипы только многомерные массивы с одинаковым кол-вом элементов в каждом вложенном массиве.
```

Comment:
No, not in a subarray. In subarrays in postgres number of elements must be the same in subarrays inside array. What I meant is that in postgres there might be arrays of different dimensions in the same column but in different rows. For example, a column type is Integer[], the first row can contain 2-dimensional array, and the second row 3-dimensional array. But it ClickHouse arrays in all rows must be of the same dimension.

Reply:
>  But it ClickHouse arrays in all rows must be of the same dimension.
@kssenii 
could you explain? 
what exactly you mean when say "2-dimensional array" it is `Array(Array(UInt32))` and "3-dimensional array" is `Array(Array(Array(UInt32)`?

if yes, why you disagree with russian term "вложенные массивы"? and use "rows" term which could be interprets as "diffrenet rows in table"?

Reply:
ok. i understood
on postgresql we just can define
```
 CREATE TABLE test_array(a BIGINT[]); 
  INSERT INTO test_array VALUES('{{1},{2},{3}}'),('{{1},{2}}');
 ```
 and insert 2 dimensional array instead of single dimension

Reply:
fixed on https://github.com/ClickHouse/ClickHouse/pull/21078/commits/b586b44169a9e83a0acb6608060ebf857b03cc8e